pkgs/development/interpreters/python/tests/test_nix_pythonprefix/typeddep/default.nix

```diff
@@ -0,0 +1,11 @@
+{ buildPythonPackage }:
+
+
+buildPythonPackage {
+
+  pname = "typeddep";
+  version = "1.3.3.7";
+
+  src = ./.;
```

Comment:
not very important but this will include this nix file.

Reply:
I figured it didn't really matter and I didn't want any source filtering obscuring anything.